src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfCreator.java

```diff
@@ -34,9 +37,30 @@ public PdfCreator(DuplexPreparator duplexPreparator, IHtmlToPdfConverter convert
         return PdfMerger.mergeDocuments(docs);
     }
 
+    public byte[] create(LetterWithPdfsRequest letter) {
+        Asserts.notNull(letter, "letter");
+
+        List<byte[]> docs =
+            letter.documents
+                .stream()
+                .map(this::decodePdf)
+                .map(duplexPreparator::prepare)
```

Comment:
There's some duplication here with the other method, we could perhaps have a helper method that calls the duplexPreparator and merges the docs.